csp/gitprojectsettings.csp

```diff
@@ -550,6 +550,35 @@ body {
             </div> -->
         </div>
 
+        <div class="form-group row mb-3">
+            <label for="addToFav" class="offset-sm-1 col-sm-3 col-form-label"  data-toggle="tooltip" data-placement="top" title="Favorite namespaces for user. For Windows/Mac: Hold down the control(ctrl)/command(cmd) button to select multiple options.">Favorite Namespaces</label>
+            <div class="col-sm-7">
+                <select multiple class="form-control" id="addToFav" name="favNamespace">
+                <server>
+                    set contextList = ##class(SourceControl.Git.Utils).GetContexts(1)
```

Comment:
Idea: you could have SourceControl.Git.Utils return the set of contexts as a byref local array, that would let you take a more sensible approach below.